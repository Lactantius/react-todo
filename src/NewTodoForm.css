.NewTodoForm {
  display: flex;
  gap: 1rem;
  align-items: center;
}

.NewTodoForm label {
  font-size: 1.2rem;
}

.NewTodoForm input {
  font-size: 1.2rem;
}

.NewTodoForm button {
  font-size: 1.2rem;
  color: white;
  background-color: #1448cc;
  border: none;
  border-radius: 5px;
  padding: 0.5rem;
}
