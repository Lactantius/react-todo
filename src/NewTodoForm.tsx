import React from "react";
import "./NewTodoForm.css";

function NewTodoForm(): JSX.Element {
  return <div className="NewTodoForm"></div>;
}

export default NewTodoForm;
