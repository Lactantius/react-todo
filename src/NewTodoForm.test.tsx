import React from "react";
import { render } from "@testing-library/react";
import NewTodoForm from "./NewTodoForm";

it("renders", () => {
  render(<NewTodoForm add={() => null} />);
});

it("matches snapshot", () => {
  const { asFragment } = render(<NewTodoForm add={() => null} />);
  expect(asFragment()).toMatchSnapshot();
});
