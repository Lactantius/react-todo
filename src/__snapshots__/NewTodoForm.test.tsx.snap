// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <form
    class="NewTodoForm"
  >
    <label
      for="text"
    >
      Todo: 
    </label>
    <input
      id="text"
      name="text"
      placeholder="What needs to get done?"
      type="text"
      value=""
    />
    <button>
      New Todo
    </button>
  </form>
</DocumentFragment>
`;
